lib/video_queue/message_consumer.rb

```
@@ -1,11 +1,15 @@
 class MessageConsumer
-  def initialize(queue)
+  def initialize(queue, processor=nil)
```

But then you would need to know how it is constructed.  Maybe change it to not take a default and that way it will blow up if it is not passed in?  Fail fast?